modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/bookmark/components/UpsellView.kt

```diff
@@ -26,17 +30,44 @@ import au.com.shiftyjelly.pocketcasts.ui.R as UR
 @Composable
 fun UpsellView(
     style: MessageViewColors,
+    activeTheme: Theme.ThemeType,
     onClick: () -> Unit,
     sourceView: SourceView,
     modifier: Modifier = Modifier,
 ) {
     val viewModel = hiltViewModel<PlusUpsellViewModel>()
+    Content(
+        style = style,
+        activeTheme = activeTheme,
+        onClick = {
+            viewModel.onClick(sourceView)
+            onClick()
+        },
+        modifier = modifier,
+    )
+}
+
+@Composable
+private fun Content(
+    style: MessageViewColors,
+    activeTheme: Theme.ThemeType,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val logoColor = if (activeTheme.darkTheme) Color.White else Color.Black
+    val description = stringResource(LR.string.pocket_casts_patron)
     MessageView(
         titleView = {
             Row(
-                verticalAlignment = Alignment.CenterVertically
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier
+                    .clearAndSetSemantics { contentDescription = description }
             ) {
-                TextH20(text = stringResource(LR.string.pocket_casts))
+                Image(
+                    painter = painterResource(IR.drawable.logo_pocket_casts),
+                    contentDescription = null,
+                    colorFilter = ColorFilter.tint(logoColor),
+                )
                 Spacer(modifier = Modifier.width(8.dp))
```

Comment:
Just wanted to confirm that we don't want the Pocket Casts icon here. If we do want that included, we could reuse the `HorizontalLogo` composable.

Even if we don't want to reuse that, it might be worth making the `Image` part of this a reusable component in the [same folder as the other logo components we have](https://github.com/Automattic/pocket-casts-android/tree/main/modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/images). That's certainly not a must-have though.

Eventually, we should probably add a "Logo" section to Showkase.

Reply:
We don't want the icon with the logo from what I see in Figma. 

> it might be worth making the Image part of this a reusable component in the [same folder as the other logo components we have](https://github.com/Automattic/pocket-casts-android/tree/main/modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/images). That's certainly not a must-have though.

Yep, I plan to add it to the showcase too.